.github/workflows/dependencies-and-licences.yml

```diff
@@ -0,0 +1,59 @@
+name: Auto Update - Spec
```

Comment:
Please change the name so that the workflow pop up as a separate item in GH actions